plugins/modules/packaging/language/cargo.py

```diff
@@ -0,0 +1,200 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2021 Radek Sprta <mail@radeksprta.eu>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+DOCUMENTATION = r"""
+---
+module: cargo
+short_description: Manage Rust packages with cargo
+version_added: 4.1.0
+description:
+  - Manage Rust packages with cargo.
+author: "Radek Sprta (@radek-sprta)"
+options:
+  name:
+    description:
+      - The name of a Rust package to install.
+    type: list
+    elements: str
+    required: true
+  path:
+    description:
+      ->
+      The base path where to install the Rust packages. Cargo automatically appends
+      C(/bin). In other words, C(/usr/local) will become C(/usr/local/bin).
+    type: path
+  version:
+    description:
+      ->
+      The version to install. If I(name) contains multiple values, the module will
+      try to install all of them in this version.
+    type: str
+    required: false
+  state:
+    description:
+      - The state of the Rust package.
+    required: false
+    type: str
+    default: present
+    choices: [ "present", "absent", "latest" ]
+requirements:
+    - cargo installed in bin path (recommended /usr/local/bin)
+"""
+
+EXAMPLES = r"""
+- name: Install "ludusavi" Rust package.
```

Comment:
```suggestion
- name: Install "ludusavi" Rust package
```
As opposed to (option) descriptions, we usually don't have a period at the end of task names :) (same below)

Reply:
Done.